### PR TITLE
travis -> github actions

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -14,6 +14,6 @@ let
     ];
   };
   packages = pkgSet.config.hsPkgs;
-in packages.cachix.components.exes.cachix  // {
+in packages.cachix.components.all  // {
```
Comment (roberth) [inline default.nix]: I suppose this isn't public api.

Comment (domenkozar) [reply default.nix]: Yes it is: https://input-output-hk.github.io/haskell.nix/reference/library/#component

Comment (roberth) [reply default.nix]: Oh I meant cachix `/default.nix`

Comment (domenkozar) [reply default.nix]: Kind of - need to fix readme I guess.

**Pull request merged**